### DISCUSSION TITLE
Is there a way to skip backtick command substitution

### DISCUSSION THREAD

**Discussion by ecliffdaddy** [mvdan/sh#1071] [Q&A] [answered]
I really like this tool,  but the result is converting backticks to $() .  I personally feel that is fine and would not even need testing.  My colleagues are not interested in converting them and said all scripts would need re-testing.  Is there an option to skip that conversion?

Thanks
Eric

**Comment by mvdan** [comment, marked as answer]
Sorry but there isn't, and I don't see us adding an option just for this. For some context, the tool has existed for many years at this point, and so far noone else has really brought up this concern.

**Comment by ecliffdaddy** [comment]
Your answer confirms my findings as well.   Thank you.
